### DISCUSSION TITLE
How to change the default UI ip port from 12345 to what i want?

### DISCUSSION THREAD

**Discussion by gzqcason** [apache/dolphinscheduler#11561] [Q&A] [answered]
By using Dolpin 2.0.3 I can change the port in the install_config.conf file, however I can’t find it in the 3.0.0 Version. Does it changed in the xx/aip-server/conf/application.yaml? BTW, what should I change when I need debug the function. TKS a lot

**Comment by DarkAssassinator** [comment, marked as answer]
u can find it in $DOLPHINSCHEDULER_HOME//api-server/conf/application.yaml

**Reply by gzqcason** [reply]
yeah，that's it，found in the official document. Tks.
Can I also change the password of the admin account in this configuration file?
a little different between version 2.0.3 , make me confuse :(
![image](https://user-images.githubusercontent.com/79817470/185856999-b1f5616f-21a3-425c-9214-ee190a126848.png)
